classes/ThankYous/Api.php

```diff
@@ -806,9 +937,6 @@ public function Notify(ThankYou $thank_you)
 				}
 				$this->line_manager_notifier->SendMessage($description, $user_ids);
 			}
-		} catch (ThankYouException $exception)
-		{
-			throw $exception;
 		} catch (Exception $exception)
 		{
 			throw new LogicException("Unexpected Exception thrown by NotificationMessage library", null, $exception);
```

Comment:
I thought we'd agreed to remove this and just let the exception bubble, or just catch and log instead of just changing the exception type unnecessarily.

Reply:
Fixed in f094fe0d8021f7fa46e7397b31a34076c1183ca7